### ISSUE TITLE
代码运行时错误，pytorch版本不对？

### ISSUE THREAD

**Issue by hwlll** [JaidedAI/EasyOCR#333]
运行easyocr.py时报错：set_storage is not allowed on Tensor created from .data or .detach()，在modules.py第87行，请问如何解决？

**Comment by rkcosmos** [comment]
sorry, I can't read Chinese.

**Comment by hwlll** [comment]
> sorry, I can't read Chinese.

sorry, when I run the easyocr.py, an error is reported: set_storage is not allowed on Tensor created from .data or .detach(), on line 87 of modules.py, how can I solve it?I searched for possible pytorch version problems, do I need a version above 1.1.0？
